users/users.md

```diff
@@ -116,6 +117,56 @@ Response
 ```
 ---
 
+## Get a list of user groups that a user belongs to
```

Comment:
Make it simpler "Get a list of user groups the user belongs to"

Reply:
updated